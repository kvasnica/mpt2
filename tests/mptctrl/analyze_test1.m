function display_test1

load ctrl1
analyze(ctrl)
