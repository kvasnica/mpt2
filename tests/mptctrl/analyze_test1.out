
The controller can be simplified:
   ctrl = mpt_simplify(ctrl)  will reduce number of regions.
